packages/flutter_test/lib/src/goldens.dart

```diff
@@ -72,6 +72,23 @@ abstract class GoldenFileComparator {
   /// is left up to the implementation class.
   Future<void> update(Uri golden, Uint8List imageBytes);
 
+  /// Can be used to set an acceptable tolerance level, establishing an
+  /// allowable difference between pixels.
+  ///
+  /// Must be between 0.0 and 1.0. For example, a tolerance of 0.01 would allow
+  /// for golden file images under test to have a 1% difference.
+  double get precisionTolerance => _precisionTolerance;
+  double _precisionTolerance = 0.0;
+  /// Sets an allowable percentage of difference between golden file images.
+  set precisionTolerance(double value) {
```

Comment:
@goderbauer I made this getter/setter instead, it has ramifications for subclasses like the regular and web `Trivial*` comparators.. Not sure which is better.